packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json

```diff
@@ -0,0 +1,6970 @@
Rt2S0XgMY2wckk7pPgZPR6plrF6wkHsUan6XTRzylKHI9x0KtlQh1ItFxglpEAiTqOc4mdy93wbaJyvSfJcfd6f2c8sryfU9Ldy2r30S2zkCCgBmqSkzzvTq3FlFrBIFQm9vDYMcyOS8bxjWcdOOmv8vojWEMuZ+YWgzzk+AC8eHDgenGkjbo1wpte5w+80tPAxh3BZakpOSUTLUl+tJGKu1zHEEQQdREEcRs7VvKNcGW0oy9C9CoNaxknyLyQKlGXhrASXRAGMknUAPgq07kq6nLrUj9W0Bo80bPTpEesAS4DNxLiOyY7F9VsuludJRfM4Hbdm30YuEgTwXQ5pPiJablyPGdOLbF2zDXIe/dH3QfivK8W2lKC011+hrpRwTbPg9H6bzaqPViXdY0gbh97gIleLoNrUjXc1StUfr7ngYGpiN6+hxb4pGlpcGz8w09ZL9O9GLcew6/Nb+M7Pnpb1c4/Q8XYNbDUx6P6nyzRJY2rGBkO3OBiTxwPGV4G2aDlpx2mK4S5+jXD58z1dPUS1Hp/wBr3f8AX0J0paZGBG0LzW7N7xZ6rR1YvYHHXq4xtX3PhGvPV2WMp8+KvvR4HiWnGGt+nqrNrAvSZwxNDGqGbRLNChmyMztF0SZNMTM4SO4GFwT8M2WcsnBX8PkuButefI+lTAjfsyXW125DTT4vmfFs3RxzrU0NbFNzrxdIgCSSAMDsiMV8pt/hjev+hfpfFv8AOR17PHKSs9/R0VRAJm9hBk4L1Z7RqP8ATyR1x0NJceZ+bWvoY6+erqs6suOLpvNbOUYn47l5kvB9XLg1RjvIVzPS6S6K0q5aWm6Q1rZEYhogXhtOAXZtGwaOrxlafddRShm+B9Do70Lo0v8AUd679l+PV4Abd654bLpaErhxfd/Y6NLRVWz7WntIijRqFgaXhjiGkSCQ0nEZLbd6koOS6Jl6usoLhzPwV8Fzo2kxAgRrAC+fm7dmE26VkbbaNTXQS0Nadc4A9mF4jYr09Pm/z84HXp0o8ehSxVu74LPUiaNWrPp6JsYqVDTotmfWOwb8cgjT0NbXkori/wA4nna6blfQ/QND6LFBpEy533jw1AbhjzX1Xh+wrZYVdt83/oxnK2bivQM7BKdE2cCgLGakUmeV6dWtjmizwC7BxdtZkBx2r5rxjar1FpR6c/76fnobxk4qzylPRDWtFSo4lpgjCC6cs9i8p6kuh0x1G1x4FNGWVr6hBaLtJrqrmnEFrG3gDxN0dq32OGepb5JNv+hXxSXVnyqjjUvPcbxJJcTtnH64qZzk5W+Z1cK4DMoNuEibwx+GHxWbnLLjyIfLgamWcRfbt1naCsnN3izk1oyZ6Tovpmo2o2lUcXMd6onEtJ1Qcty9Xw/b5x1Fpydp8Pccd48Ue6pW64IAC+ilo5cWzeOvjwSPzi1aAtFWtUqVLsucSXE4GcgJIG7cvF1PDNp1dRt1V87Hq68Luz7OhtHCzS5pmoRF6IgZNyXq7H4Zp6CuXFnJPaZX+ngXe8kySvUUUkcj1G3bZguyIOo4LWUVKLi+TOW6doy6LogMdTIkNe5pnaCAfgQuDw/QUdnejNXTafrxOva9RuUNRPmv9sq3RdL3eZJCmPg2xxllj83QPxLXaq18DaxgAgCBkvSjGMUoxVJHDOcpO5O2WY1NhEu0KGbIoEjRMYJFWOkVYWuISasalRobbH3boMDcs3pRvJo1WvKsU+BMFVRORRpUtFqRtoW9zdpKwnoRkdENokjPa3Gpen2mkcxCrdpQxREpuTtn5bZtHuqF5bqpgvecg3X2k4L47Q0XqSa5JK2dknSXuPm2qkC91ybs7dp2mOKHJJnZpp0NRo+y4cCspS6od9DdYa9Si4PpmSNWfAhKGq4TU48GjDVjaP0rRWlmVwbshwi806xvGYX1mx7dp7Uv08GuaPMs2ldwrAmKwhILI2+1ilTdUPsieJ2DtK59p1lo6UtR9Pr0LjxZ4Rliq1W1LQ8Ej7znHCS4hsNz19kL4x6WrqxlrPkuLfd30/s6Oqsx2sOcAS4E4ADY1o2DIYLBSRo/Q9JW0KaVhe5uNV7WOqHaKY9YsHDAnOOC+gjsT09jk4/uat+7nQtKdz+h4Wi26S3YcF48naTO6DL2EG9EScRGc4HsiVnq1VkydXZ9vRehaxL+rbfaIOESJnWD4ZKYaeptKvTjxXMxnK6PraJ6PVG1GVHtuta69BImRqwG9et4f4br72OpqKkvicmtjR6i7K+o5GHMzVArRlIx1nLSKMJyMhetKOd6hnaFoZtjtaiiXJscBMlsdrUhFmtSZoigCk0RQBItBASKTLOfOwDgpSo1c76EyqM2wtCTGmUaEi0ygUlpjBIaY7VJaZ8K26P6mlWewOe58BrWtOAvA7Neocl4UvDvLx1dSNycrpJck3Z0wmpNI8M6ykEAzJOIjGSY1ZzhC+fuUpNVx7ep6MpJcj7lTotaQ4tuhwB+8HCCM4Jnshdut4XtEG6jaXW1/wCmK1otcWZ6NjhxgYRJOWIAPMjmvHycoNrp9BXaaN+iCW16Zgg3g05G96pnmurw3WentUGurS+PA5tSHA9svvjlsCBWEBJjR1SkCIcARkYKmSjJU1ZabXIL6QLS0j1SII3ZKZwjOLhJcHwGpNOz5Vl6O02vLj6w9lp1Dj7y8jR8F0NPUc3xXRPp7+/5dmr1m1SPtFeuZ2ed0t0Ro1fWp/6Tv7RLTxb5QvP1/DdHUX6f0v0+xvHaGuZlo9DWsAIqFz5GJ9VoGMkASSe1cOr4LenjGXG+b5V7i3tOXNHodH2QUWXGkmTLjmfJensWww2XTxjxfV9zmlqNso8rtSM2yL1aM2zPUVIylIx1pWsTmnJmQhWYWBoVA2OGoJscNQTZRjUhrmXDVJsMGpDHAQWjoSGNCB2ddQAwakUO0KSkx4SKsISY0xwkWmMEh2Zxo2l1nXXB1nvZbwNQO9YeX0t5vMVl3Nd5Kqs2ArVqyVIxWbRVNnWYE9YC07gTMDu
```

Comment:
A value of never means that after every attempt, the student will only be able to see if an answered question is incorrect.